presentations/index.md

```diff
@@ -1,10 +1,13 @@
+This is a collection of Gluster presentations from all over the world. 
+We have a [slideshare account](http://www.slideshare.net/GlusterCommunity) where most of these presentations are stored. 
+
```

Comment:
Could you add a note how people can get their presentations added to that account? What do presenters need to do to have their presentations listed on this page with a slideshare link?